index.html

```diff
@@ -0,0 +1,264 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio: finish mobile versio</title>
+    <link rel="stylesheet" href="./css/styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <div class="container">
```

Comment:
- [ ] kindly bring back the home page, it should be a part  of  your portfolio with the design match to the Figma design :grinning: :grinning: :grinning:
## your design
![image](https://user-images.githubusercontent.com/95297251/185766851-00f1553d-70c2-4ff9-a049-3146ab644aa7.png)
## Figma design
![image](https://user-images.githubusercontent.com/95297251/185766888-a468ca4c-8c3b-4eca-ab24-d025e2c3b35f.png)